### ISSUE TITLE
[question] libalsa: shouldn't it be a system package?

### ISSUE THREAD

**Issue by nicosmd** [conan-io/conan-center-index#17101]
### What is your question?

Hey,
 
I'm curious about why libalsa wasn't made a system package but instead a normal package, which builds from source. From my understanding the library itself might be compatible with different linux kernels but at least the configuration and plugins should be read from the system rather than from the conan package. 

The current libalsa conanfile has in my opinion some problems when using it. In order to make it run properly I had to set ALSA_PLUGIN_DIR and ALSA_CONFIG_DIR manually to the system directories. Otherwise I get a Segmentation Fault. 
This might be an indicator that a system package might be better suited, or am I missing something?

Thanks for your answers!

**Comment by jwillikers** [comment]
Yeah, we need to address this. I think this is a duplicate of  #2474.

**Comment by nicosmd** [comment]
Closed as duplicated
